sidekick/sidekick-statistics/sidekick-statistics-service/src/main/java/com/runsidekick/service/impl/ServerStatisticsServiceImpl.java

```diff
@@ -0,0 +1,93 @@
+package com.runsidekick.service.impl;
+
+import com.runsidekick.model.ServerStatistics;
+import com.runsidekick.repository.ServerStatisticsRepository;
+import com.runsidekick.service.ServerStatisticsService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * @author yasin.kalafat
+ */
+@Service
+public class ServerStatisticsServiceImpl implements ServerStatisticsService {
+
+    private static final Logger LOGGER = LogManager.getLogger(ServerStatisticsServiceImpl.class);
+
+    @Value("${phonehome.statistics.enabled:true}")
```

Comment:
I would autowire and use `PhoneHomeConfig` here to not to duplicate property name around the code